### PR TITLE
release: Amplify Android 1.17.4

### PR DESCRIPTION
**Pull request by awsmobilesdk**
## Release PR review checklist
- [x] Verify version name in gradle.properties
- [x] Verify CHANGELOG.md

## Module versions

Amplify Android 1.17.4
Amplify Android 0.1.5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## [Release 1.17.4](https://github.com/aws-amplify/amplify-android/releases/tag/release_v1.17.4)
+
+### Bug Fixes
+- Use ObjectsCompat since Objects is API 19+ (#1289)
+
+### Miscellaneous
+- depend on latest sdk version (#1294)
```
Comment (changxu0306) [inline CHANGELOG.md:7]:
```suggestion
- Depend on the latest SDK version (#1294)
```

**Review by raphkim — APPROVED** [review]
Looks good. Can we do a manual retry with CodeBuild unit test to make sure before we merge though?

**Comment by changxu0306** [comment]
> Looks good. Can we do a manual retry with CodeBuild unit test to make sure before we merge though?

CodeBuild unit test retry was successful. I think it's safe to merge.

**Review by richardmcclellan — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## [Release 1.17.4](https://github.com/aws-amplify/amplify-android/releases/tag/release_v1.17.4)
+
+### Bug Fixes
+- Use ObjectsCompat since Objects is API 19+ (#1289)
+
+### Miscellaneous
+- Depend on the latest SDK version (#1294)
```
Comment (changxu0306) [inline CHANGELOG.md:7]:
```suggestion
- Depend on the latest SDK version (#1294, #1296)
```

File: core-kotlin/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
+## [Release 0.1.5](https://github.com/aws-amplify/amplify-android/releases/tag/release-kotlin_v0.1.5)
+
+### Bug Fixes
+- Use ObjectsCompat since Objects is API 19+ (#1289)
+
+### Miscellaneous
+- depend on latest sdk version (#1294)
```
Comment (changxu0306) [inline core-kotlin/CHANGELOG.md:7]:
```suggestion
- Depend on the latest SDK version (#1294, #1296)
```

**Pull request closed without merging**